pkg/controller/persistentvolume/framework_test.go

```diff
@@ -372,8 +375,8 @@ func checkEvents(t *testing.T, expectedEvents []string, ctrl *PersistentVolumeCo
 				glog.V(5).Infof("event recorder finished")
 				finished = true
```

Comment:
The entering for loop condition is &, so what is the difference between the old version and new one if the path comes to this point? After finished is set true at this line, it will quit waiting, I think.